### DISCUSSION TITLE
Hacktoberfest 2022

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/virtualcoffee.io#612] [Monthly Challenge]
Hey friends! 

We're heading into our favorite time of year: Hacktoberfest.

We love organizing an event where many of our VC members can participate and feel supported. As in years past, we'll provide support for maintainers and contributors. 

But we'd love to make sure this is an event that continues to provide value to the community. That's why we're turning to you. If you have ideas for special things we can do for Hacktoberfest this year, we want to hear them. Drop them in the discussion below. (We'll have volunteer sign-ups at a later date. Right now, we want to focus on brainstorming how to make this the best event of the year.)

**Comment by Cerchie** [comment]
What about a couple of 'mob programming' sessions to work on website tix? Happy to spearhead.

**Reply by Cerchie** [reply]
The con of this idea is, then only 1 person gets Hacktoberfest credit, so maybe this can be part of Preptember instead, as a PR lesson etc.

**Reply by dominicduffin1** [reply]
@Cerchie That could be a great idea for Preptember, maybe we could have mob sessions for preparing projects to accept contributions or other work that maintainers want to complete prior to Hacktoberfest?

**Reply by Andrew-Bush** [reply]
I think this would really go far in helping expose devs to what to the process of contributing, without the weight of it having to be in their names.

**Comment by JesseRWeigel** [comment]
I did a hacktoberfest video a few years back. I could update it. Also up to do some livestreams.

**Comment by BekahHW** [comment]
Last year's Hacktoberfest: https://virtualcoffee.io/monthlychallenges/oct-2021

Preptember: https://virtualcoffee.io/monthlychallenges/sept-2021
